csm/core/services/maintenance.py

```diff
@@ -67,10 +67,13 @@ async def get_status(self) -> Dict:
         """
         Return status of cluster. List of active and passive node
         """
-        Log.debug("Get cluster status")
+        Log.info("Get cluster status")
         try:
-            return await self._loop.run_in_executor(self._executor,
+            node_info = await self._loop.run_in_executor(self._executor,
                                                 self._ha.get_nodes)
+            for each_resource in node_info.get("node_status"):
```

Comment:
Use const class instead of hard coded values.

Reply:
Comment resolved. Please review